scripts/modules/proxy.py

```diff
@@ -29,7 +29,10 @@ def _content(self):
             logging=None,
             healthcheck=wget_healthcheck(8000, path="/"),
             ports=["9000:8000"],
-            volumes=["/var/run/docker.sock:/var/run/docker.sock"],
+            volumes=[
+                "/var/run/docker.sock:/var/run/docker.sock",
+                "./docker/dyno:/dyno",
```

Comment:
This doesn't look related to the styling changes. Could you clarify please?

Reply:
Sure, because I need a fast development environment, I need to edit a static file (JS, HTML) and changes be updated into the deployed application, without the need of rebuilding the compose file or the image. For that reason I'm mounting the entire application SRC into the container, achieving fast reload on the static files:

- JS and CSS are automatically reloaded, so refreshing the resource in the browser will do the trick
- HTML is tricky, as it's not directly served but processed by the Flask app. I only need to restart the container (3-4 seconds) and the HTML is live after a refresh in the browser. If you know a manner to add hot-reloading to the gunicorn process, I'm up to it.

Reply:
👍 Sounds good. Thanks!